src/users/components/user-personal-details/user-personal-details.component.html

```diff
@@ -0,0 +1,97 @@
+<h2 class="govuk-heading-l">User Details</h2>
+
+<ng-container *ngIf="inviteMode; else readOnlyTemplate">
+  <form *ngIf="personalDetailForm" [formGroup]="personalDetailForm">
+    <xuilib-gov-uk-form-group-wrapper
+      [group]="'personalDetailForm'"
+      [config]="{
+        key: 'firstName'
+      }"
+    >
+      <xuilib-gov-uk-input
+        [config]="{
+          label: 'First name',
+          hint: 'Include all middle names',
+          name: 'firstName',
+          id: 'firstName',
+          type: 'text',
+        }"
+        [group]="personalDetailForm"
+        [errorMessage]="{
+          isInvalid: errors.firstName.length > 0,
```

Comment:
Is there a limit to how long fistname, lastname and email can be or is it unlimited? Also someone can enter only 1 character and it would be valid!